Sources/SwiftMemcache/MemcachedConnection.swift

```diff
@@ -168,15 +175,15 @@ public actor MemcachedConnection {
     ///
     /// - Parameters:
     ///   - key: The key to set the value for.
-    ///   - value: The value to set for the key.
+    ///   - value: The `Value` to set for the key.
     /// - Returns: A `ByteBuffer` containing the server's response to the set request.
-    public func set(_ key: String, value: String) async throws -> ByteBuffer? {
+    public func set<Value: MemcachedValue>(_ key: String, value: Value) async throws -> ByteBuffer? {
```

Comment:
While we are at it we should probably remove the `ByteBuffer` return here. I don't think that makes much sense right now.